.github/ISSUE_TEMPLATE/organization-membership-request.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Organization Membership Request
+about: Request membership in an Istio Org
+
+---
+
+### GitHub Username
+e.g. (at)example_user
+
+### Organization you are requesting membership in
+e.g. (at)istio
+
+### Company Affiliation or Individual
+e.g. Google, IBM, VMware, Red Hat or XX etc, otherwise, Individual
+
+### Requirements
+- Subscribed to istio-dev@googlegroups.com (istio-dev@googlegroups.com).
+- Actively contributing to 1 or more areas.
+- Sponsored by 2 members.
+
+- [ ] I have reviewed the community membership guidelines (https://github.com/istio/community/blob/master/ROLES.md#member & https://github.com/istio/community/blob/master/CONTRIBUTING.md)
+- [ ] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
+- [ ] I have subscribed to the istio-dev@googlegroups.com email list
+- [ ] I have made multiple contributions to the project or community. Contributions may include, but are not limited to:
+    - Authoring or reviewing PRs on GitHub
+    - Filing or commenting on issues on GitHub
+    - Contributing to working group or community discussions
+- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines.
+- [ ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
+
+### Sponsors (Please mention sponsors’ github ids here)
+- (at)sponsor-1
+- (at)sponsor-2
+
+### List of contributions to the Istio project
+- PRs reviewed 
+- PRs authored
+- Issues responded to
+- Workgroup(s) I am involved with
+
+### Slack access:
+- What is your email address associated with Slack?
```

Comment:
Please change to:

If you'd like to join our developer Slack workspace, what is the email address you have associated with Slack?